rotkehlchen/data_import/utils.py

```diff
@@ -75,9 +117,45 @@ def flush_all(self, write_cursor: DBCursor) -> None:
         self._asset_movements = []
         self._history_events = []
 
+    def append_msg(self, row_index: int, msg: str, warning: bool) -> None:
+        """Append message to queue to be sent to frontend
```

Comment:
Format the docstrings better please.

Also expplain what the arguments are. What is `warning` more specifically
```suggestion
        """Append message to queue to be sent to frontend.

```